intelmq/bots/collectors/rsync/collector_rsync.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import os
+from intelmq.lib.bot import CollectorBot
+
+
+class RsyncCollectorBot(CollectorBot):
+    def process(self):
+        self.logger.info("Updating file {}.".format(self.parameters.file))
+        os.system("rsync {}/{} {}".format(self.parameters.rsync_path, self.parameters.file, os.path.dirname(__file__)))
```

Comment:
Please do not use `os.system`, but [`subprocess.call`](https://docs.python.org/3/library/subprocess.html#subprocess.call). The former has no security at all.